src/graph/serializers/decoders/prev/decode_previous.c

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2018-2019 Redis Labs Ltd. and Contributors
+ *
+ * This file is available under the Redis Labs Source Available License Agreement
+ */
+
+#include "decode_previous.h"
+#include "../../../../version.h"
+
+GraphContext *Decode_Previous(RedisModuleIO *rdb, int encver) {
+	switch(encver) {
+	case 4:
+		return RdbLoadGraphContext_v4(rdb);
+	case 5:
+		return RdbLoadGraphContext_v5(rdb);
+	default:
+		assert(false);
```

Comment:
we can change assert(false) to assert(msg) to give some information on this failure